indexing-service/src/main/java/io/druid/indexing/overlord/autoscaling/ScalingStats.java

```diff
@@ -65,6 +65,15 @@ public ScalingStats(int capacity)
     }
   }
 
+  public ScalingStats(ScalingStats stats1, ScalingStats stats2)
```

Comment:
IMHO it would be better to have a `addAll(Collection<? extends ScalingEvent> events)` kind of method instead of a constructor that only takes 2.